### PR TITLE
Milestone 2: Portfolio - finish mobile version

### PR DESCRIPTION
**Pull request by grauJavier**
# New Features
- Sections with **Works** and **About Myself** were added to the portfolio webpage.
- **Minor changes in** ***index.html*** related to ***tags and class/id names.***
- Re-edition and **major changes were made in** ***style.css*** incorporating ***flex*** and ***grid*** displays.
- New visuals files added, such as pictures and icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,63 +1,205 @@
...
+      </ul>
+    </div>
+  </section>
+  <section id="works">
+    <div class="work-card">
+      <img class="work-snapshot element" src="./visuals/tonic-snapshot.png" />
+      <div class="work-text-block element">
+        <h1 class="title" id="headline-title">
+          Tonic
         </h1>
```
Comment (ABDULALI3468) [inline index.html]: Please consider using a heading tag other than the h1 tag here as you have already used it in the headline section. The reason is there must be only one h1 tag through so the browser can easily understand what you want to show users when they search.


## NOTE:-

This problem exists in all the cards so please consider using other heading tags like h2 or other for all.

File: index.html [hunk 2]
```diff
@@ -1,63 +1,205 @@
...
       </div>
-    </section>
-  </body>
+    </div>
+  </section>
+  <section id="about-myself">
+    <div id="about-myself-container">
+      <h1 class="title" id="headline-title">
+        About Myself
+      </h1>
```
Comment (ABDULALI3468) [inline index.html]: ✅ Please consider using a heading tag other than the h1 tag here as you have already used it in the headline section. The reason is there must be only one h1 tag through so the browser can easily understand what you want to show users when they search.

File: index.html [hunk 3]
```diff
@@ -1,63 +1,205 @@
...
+    <ul id="boxes">
+      <li class="box" id="box-languages">Languages<img class="chevron-icon" src="./visuals/chevron-down-icon.png"/></li>
+      <ul class="box-list" id="languages">
+        <li class="box-element"><img class="language-icon" src="./visuals/javascript-icon.png"/>JavaScript</li>
+        <li class="box-element"><img class="language-icon" src="./visuals/html-icon.png"/>HTML</li>
+        <li class="box-element"><img class="language-icon" src="./visuals/css-icon.png"/>CSS</li>
+      </ul>
+      <li class="box">Frameworks<img class="chevron-icon" src="./visuals/chevron-right-icon.png"/></li>
+      <li class="box">Skills<img class="chevron-icon" src="./visuals/chevron-right-icon.png"/></li>
+    </ul>
```
Comment (ABDULALI3468) [inline index.html]: Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with a detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
      </li>
      <li>
        Skills
      </li>
    </ul>
```

## CONCLUSION:-

Kindly make the list of languages inside the li tag as presented in the above code.

File: index.html [hunk 4]
```diff
@@ -1,63 +1,205 @@
...
+          <span class="bullet">&#x2022</span>
+          <li class="work-year">2015</li>
         </ul>
+        <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-list">
+          <li class="tag">HTML</li>
+          <li class="tag">CSS</li>
+          <li class="tag">JavaScript</li>
+        </ul>
+        <a class="button" href="#">See project</a>
```
Comment (ABDULALI3468) [inline index.html:84]: Please consider using the same border for the button to make the styles exactly in accordance with the Figma design:-

### YOUR DESIGN:-

image


### FIGMA DESIGN:-

image


## NOTE:-


This problem exists in all similar buttons so also consider reviewing them to match the styles with the design.

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @grauJavier  , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights 
☑️Well implemented design.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by grauJavier** [comment]
@ABDULALI3468 Thank you very much for your review. I made the fixes you suggested. I'm all 👀 and 👂!

**Review by topeogunleye — APPROVED** [review]
### Hi @grauJavier, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Replaced extra `h1's`
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
- [ ] It seems like you have named your project branch as mobileVersion-grid-flex. While this is a descriptive name, it does not necessarily convey that the project does not use a particular feature or action.

- When naming a project branch, it's important to choose a name that accurately reflects the contents of the branch. In this case, mobileVersion-grid-flex indicates that the branch is related to mobile version, grid, and flex . However, it doesn't explicitly convey any other information about what the branch does or doesn't use e.g displaying Hello Microverse on the web page.

- I If there is a specific feature or action that your project branch does not use, it might be helpful to include that in the name. For example, if your branch finishes portfolio mobile version, you could name it added-mobileVersion-grid-flex
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**